rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -1259,6 +1260,28 @@ def test_latest_upgrade_adds_remove_tables(user_data_dir):
     assert new_views == set()
 
 
+def test_steps_counted_properly_in_latest_upgrade(user_data_dir):
```

Comment:
docstring on what this test is testing.

Also why do it only for last upgrade? Can't you do this check for all the different DB upgrades? I manually checked them during code review but making sure that steps add up for each upgrade should be possible during tests.

Reply:
Yeah, alright